test/ticTacToeLiteTest.js

```diff
@@ -0,0 +1,64 @@
+let tttLite = require('../ticTacToeLite.js')
+let helpers = require('./unitTestHelpers')
+
+describe("ticTacToeLite.js", function() {
+  this.beforeEach(function() {
+        ttt.enableLite()
+  })
+
+  describe("#addMoveToHistory", function() {
+
+    it("can add a move to the history", function() {
+        expected = 4
+
+        tttLite.addMoveToHistory(expected)
+        actual = tttLite.getRemovalIndex()
```

Comment:
Can you think of any drawbacks to making this test dependent on methods that are not themselves under test in this scope?

Reply:
Addressed in 6b78ebf